src/usdb_syncer/gui/comment_dialog.py

```diff
@@ -0,0 +1,34 @@
+"""Dialog to post a comment on one or multiple songs."""
+
+from typing import Callable, Iterator
+
+from PySide6.QtWidgets import QDialog, QWidget
+
+from usdb_syncer.gui.forms.CommentDialog import Ui_Dialog
+from usdb_syncer.usdb_scraper import RequestMethod, get_usdb_page
+from usdb_syncer.usdb_song import UsdbSong
+
+
+class CommentDialog(Ui_Dialog, QDialog):
+    """Dialog to post a comment on one or multiple songs."""
+
+    def __init__(
+        self, parent: QWidget, selected_songs: Callable[[], Iterator[UsdbSong]]
```

Comment:
Would be better to just accept a list or iterator of song ids.

Reply:
Yeah my first implementation was to accept a list, but then i made it callable so we don't have the method execution in mw.py. I can revert that, thanks

Reply:
Fixed this and it's also just a single song now.